src/engraving/libmscore/edit.cpp

```diff
@@ -3142,7 +3142,11 @@ void Score::cmdDeleteSelection()
     deselectAll();
     // make new selection if appropriate
     if (noteEntryMode()) {
-        cr = _is.cr();
+        if (cr) {
+            _is.setSegment(cr->segment());
+        } else {
+            cr = _is.cr();
+        }
```

Comment:
This emulates the MU3 "backspace" behaviour where the note entry cursor moves to the note just deleted. It will do this when using either "delete" or "backspace", but unlike MU3 adds a new action to the undo stack (which can then be undone etc.).